src/Stratis.Bitcoin.Cli/Program.cs

```diff
@@ -196,6 +210,49 @@ public static void Main(string[] args)
             }
         }
 
+        private static string CallApiGet(HttpClient client, string url, List<string> commandArgList)
```

Comment:
Good stuff.
Can you use Flurl instead of doing this manually?
We're using it in the integration tests project.